src/main/java/com/amazon/opendistroforelasticsearch/sql/query/planner/converter/SQLToOperatorConverter.java

```diff
@@ -54,11 +57,14 @@ public SQLToOperatorConverter(Client client, ColumnTypeProvider columnTypeProvid
 
     @Override
     public boolean visit(MySqlSelectQueryBlock query) {
+        //1. extract function names in select
+        List<String> selectMethodNames = extractSelectFunctionNames(query.getSelectList());
```

Comment:
Instead of just extracting the selectMethodNames, Is it possible to generate the final select name list regardless the field is method or not?

Reply:
Will take another look.

Reply:
After another look, one way to make the code cleaner is to move the name extraction function and lowercase normalization function into the SQLAggregationParser.parse logic.